mod-fqm-manager/src/main/resources/corsair/mod-fqm-manager/features/util/add-query-data.feature

```diff
@@ -63,12 +63,19 @@ Feature: Add FQM query data
 
     # Add instance
     * def instanceId = 'c8a1b47a-51f3-493b-9f9e-aaeb38ad804e'
-    * def instanceRequest = {id: '#(instanceId)', title:  'Some title', source: 'Local', instanceTypeId: '#(instanceTypeId)'}
+    * def instanceRequest = {id: '#(instanceId)', title: 'Some title', source: 'Local', instanceTypeId: '#(instanceTypeId)', languages: ['eng', 'fre']}
     Given path '/instance-storage/instances'
     And request instanceRequest
     When method POST
     Then status 201
 
+    # Wait until last instance is indexed
+    Given path '/search/instances'
+    And param query = 'cql.allRecords=1'
+    And retry until response.totalRecords == 1
+    When method GET
+    Then status 200
+
```

Comment:
needed to ensure the instances get indexed, otherwise we have a race condition with mod-search